packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -104,9 +106,11 @@ library TaikoData {
         bytes32 metaHash;
         bytes32 parentHash;
         bytes32 blockHash;
+        bytes32 blobHash;
```

Comment:
I think we can simply pass both "Metadata" and "Evicence" to the verifier, and remove duplicate fields, such as `blobHash` and `usingBlob` from Evidence, we can also delete `metaHash` from Evidence.